pkcs12.go

```diff
@@ -73,10 +73,15 @@ type UnsupportedFormat string
 func (e UnsupportedFormat) Error() string { return string(e) }
 
 // ConvertToPEM converts all "safe bags" contained in pfxData to PEM blocks.
-func ConvertToPEM(pfxData []byte, password string) (blocks []*pem.Block, err error) {
-	p, err := bmpString(password)
-	password = ""
+func ConvertToPEM(pfxData []byte, utf8Password []byte) (blocks []*pem.Block, err error) {
```

Comment:
`utf8.DecodeRune` will fail if the underlying encoding is anything else than utf8. Indicating that the byte slice contains uft8 seems useful.